src/bastion-executor/benches/blocking.rs

```diff
@@ -15,19 +12,17 @@ use test::Bencher;
 #[bench]
 fn blocking(b: &mut Bencher) {
     b.iter(|| {
-        let handles = (0..10_000)
+        (0..100)
             .map(|_| {
                 blocking::spawn_blocking(
                     async {
-                        let duration = Duration::from_millis(1);
+                        let duration = Duration::from_millis(0);
```

Comment:
This doesn't wait at all, also why do we need that?

Reply:
I was only benching the spawn, but i ll revert!